README.md

```diff
@@ -1 +1,285 @@
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Use git clone command or download ZIP folder 
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:tsheporamantso/Hello_World.git
+```
+
+
+### Install
+
+Install this project with:
+
+npm
+
+Example command:
+
+```sh
+  cd my-project
+  npm init -y
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+npm start or live server
+
+
+Example command:
+
+```sh
+  GitHub Pages Server
+```
+
+
+### Run tests
+
+To run tests, run the following command:
+
+npm test
+
+
+Example command:
+
+```sh
+  npx stylelint "**/*.{css,scss}"
+```
+
+
+### Deployment
+
+You can deploy this project using:
+
+GitHub Pages 
+
+
+Example: 
+
+```sh
+https://tsheporamantso.github.io/Hello_World/
+```
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Gladwin Tshepo Ramantso**
+
+- GitHub: [@tsheporamantso](https://github.com/tsheporamantso)
+- Twitter: [@ramgt001](https://twitter.com/ramgt001)
+- LinkedIn: [Tshepo Gladwin Ramantso](https://www.linkedin.com/in/tshepo-ramantso-b6a35433/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [https://github.com/tsheporamantso/Hello_World/issues](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+
+If you like this project please follow me on github & twitter and also connect on Linkedin.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse for this exercise. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```

Comment:
- [ ] Dear @tsheporamantso, You did am impressive job, I just have this comment. Please remove this section from your Readme file, this is just to give you direction on how to make use of the `MIT` file and should no more be part of your readme file.

Reply:
Thank you for the feedback implemented the changes as recommended.